server/data/mappers/timelineMapper.test.ts

```diff
@@ -14,7 +15,7 @@ describe('timelineMapper', () => {
           eventType: 'ACTION_PLAN_CREATED',
           prisonId: 'MDI',
           actionedBy: 'RALPH_GEN',
-          timestamp: '2023-09-01T10:47:38.565Z',
+          timestamp: moment('2023-09-01T10:47:38.565Z').toDate(),
```

Comment:
Ah; _this_ line needs to stay as being a string. Its the value from the API (ie. the Response object) and will always be a string). It's only "our" objects (so in this case the view model object) that we can use a richer type with)